packages/insomnia-app/app/ui/components/codemirror/code-editor.js

```diff
@@ -759,8 +759,8 @@ class CodeEditor extends React.Component {
 
   _codemirrorValueBeforeChange(doc, change) {
     const value = this.codeMirror.getDoc().getValue();
-
-    if (value === '') {
+    // Suppress lint on empty doc or single space exists (default value)
+    if (value === '' || value === ' ') {
```

Comment:
Sorry I broke this yesterday! Instead of checking default, can we use this to ignore any amount of whitespace instead?

```suggestion
    if (value.trim() === '') {
```

Reply:
Ahh nice nice, yes!

Reply:
Ahh nice nice, yes!